presto-geospatial/src/test/java/com/facebook/presto/geospatial/TestGeoQueries.java

```diff
@@ -74,6 +75,29 @@ public void testSTArea()
+
+        // empty geometry
+        assertFunction("ST_Buffer(ST_GeometryFromText('POINT EMPTY'), 1)", GEOMETRY, null);
+
+        // negative distance
+        assertInvalidFunction("ST_Buffer(ST_Point(0, 0), -1.2)", "Distance parameter for ST_Buffer must be greated then or equal to 0");
+    }
```

Comment:
please add tests for the cases where `distance` is: `infinity()`, `-infinity()`, and `nan()`.